src/main/kotlin/no/nav/familie/ks/sak/kjerne/avstemming/domene/KonsistensavstemmingKjøreplan.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.familie.ks.sak.kjerne.avstemming.domene
+
+import java.time.LocalDate
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.EnumType
+import javax.persistence.Enumerated
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.SequenceGenerator
+import javax.persistence.Table
+
+@Entity(name = "KonsistensavstemmingKjoreplan")
+@Table(name = "konsistensavstemming_kjoreplan")
+class KonsistensavstemmingKjøreplan(
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "konsistensavstemming_kjoreplan_seq")
+    @SequenceGenerator(name = "konsistensavstemming_kjoreplan_seq")
+    val id: Long = 0,
+
+    @Column(name = "kjoredato", nullable = false)
+    val kjøreDato: LocalDate,
```

Comment:
```suggestion
    val kjøredato: LocalDate,
```

Enten kjører vi på med kjøredato (med liten D) eller stor D, men da ønsker jeg endring av kolonnenavn til kjore_dato